libs/elements/layout/images/src/lib/components/elewa-image-component/elewa-image-component.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="elewa-image-container">
+  <div class="elewa-images" [id]="imageVisualisation(imageConfig) === 'window' ? 'window-card' : ''" >
+    <img [src]="imageConfig.imageSrc" alt="{{ imageConfig.title }}" [ngClass]="imageVisualisation(imageConfig)"
+      [style.max-width]="imageConfig.maxWidth ? imageConfig.maxWidth : 'none'" />
+  </div>
```

Comment:
?? Describe what is happening here. Also, most of the heavy logic in your template should be handled in your ts file.